<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>bwEnc</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tblFileList">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblInputEncoding">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Input encoding:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboInputEnc"/>
      </item>
      <item>
       <spacer name="hspcr1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblOutputEncoding">
        <property name="text">
         <string>Output encoding:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboOutputEnc"/>
      </item>
      <item>
       <spacer name="hspcr2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnConvert">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="actionRemoveFiles"/>
    <addaction name="actionClearList"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionMaximumFileSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFiles"/>
   <addaction name="actionAddFolder"/>
   <addaction name="actionRemoveFiles"/>
   <addaction name="actionClearList"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add files...</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="text">
    <string>Add folder...</string>
   </property>
   <property name="toolTip">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
   <property name="toolTip">
    <string>Clear list</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionMaximumFileSize">
   <property name="text">
    <string>Maximum file size...</string>
   </property>
  </action>
  <action name="actionRemoveFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove files</string>
   </property>
   <property name="toolTip">
    <string>Remove files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
